pkg/types/vm.go

```diff
@@ -20,8 +20,8 @@ type VM struct {
 
 	EstimatedCost int64 `json:"estimated_cost"` // Represents estimated credits for this vm based on the TTL
 
-	OverlayEndpoint string `json:"overlay_endpoint,omitempty"`
-	OverlayToken    string `json:"overlay_token,omitempty"`
+	SSHPort     int64  `json:"ssh_port,omitempty"`
```

Comment:
i understand now.  we should maybe rename this to "direct ssh port" or something, since we will have a more supported ssh endpoint via CMX forwarder